build.gradle

```diff
@@ -66,6 +66,7 @@ apply from: 'gradle/forbidden-dependencies.gradle'
 apply from: 'gradle/formatting.gradle'
 apply from: 'gradle/local-distribution.gradle'
 apply from: 'gradle/fips.gradle'
+apply from: 'gradle/run-with-fips.gradle'
```

Comment:
this script allows to configure/build/deploy/run testcluster with included FIPS libraries.